frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+            </p>
+          </div>
+
+          <div>
+            <div className="flex gap-2 mb-1 items-center">
+              <h3 className="text-sm font-medium">Automated Channels Backup</h3>
+              {info?.albyAccountConnected ? (
+                <Badge variant={"positive"}>Active</Badge>
+              ) : (
+                <Badge>Recommended</Badge>
+              )}
+            </div>
+            {info?.albyAccountConnected ? (
+              <>
+                <p className="text-muted-foreground text-sm mb-8">
+                  Your channel state is backed up automatically after each
+                  channels creation. Potential recovery will trigger channel
+                  closures, and your funds will arrive in your on-chain balance.
+                </p>
+                {info?.vssSupported && (
+                  <>
+                    <div>
+                      <div className="flex gap-2 mb-1 items-center">
+                        <h3 className="text-sm font-medium">
+                          Dynamic Channels Backup With Instant Recovery
+                        </h3>
+                        {me?.subscription.buzz && info.ldkVssEnabled ? (
+                          <Badge variant={"positive"}>Active</Badge>
+                        ) : (
+                          <Badge>Alby Cloud</Badge>
+                        )}
+                      </div>
+                      <p className="text-sm text-muted-foreground mb-4">
+                        When enabled, your channels state is dynamically updated
+                        on a virtual disk encrypted with your wallet keys
+                        (Versioned Storage Service). This allows you to
+                        immediately recover or migrate your Hub with full
+                        spending balance and without having to close your
+                        channels.
+                      </p>
+
+                      {!me?.subscription.buzz ? (
+                        <Button
+                          variant="secondary"
+                          disabled={info.ldkVssEnabled}
+                          size={"lg"}
+                          onClick={() => {
+                            if (!me?.subscription?.buzz) {
+                              openLink("https://getalby.com/subscription/new");
+                            }
+                          }}
+                        >
+                          Enable Dynamic Channels Backup
+                        </Button>
+                      ) : (
+                        <AlertDialog>
+                          <AlertDialogTrigger asChild>
+                            <LoadingButton
+                              variant="secondary"
+                              loading={isMigratingStorage}
+                              disabled={info.ldkVssEnabled}
+                              size={"lg"}
+                            >
+                              Enable Dynamic Channels Backup
```

Comment:
This button shows even when VSS is enabled, I think it should not be shown.

![image](https://github.com/user-attachments/assets/055ba9ba-96fe-4f22-915e-ca5f118250ac)